components/org.wso2.carbon.identity.oauth.ui/src/main/resources/web/oauth/edit.jsp

```diff
@@ -832,6 +834,14 @@
                                         </table>
                                     </td>
                                 </tr>
+                                <tr id="authorizationCodePlain">
+                                    <td class="leftCol-med"><fmt:message key='authorization.code.validity.period'/></td>
+                                    <td><input id="authorizationCodeValidityPeriod" name="authorizationCodeValidityPeriod"
+                                               type="text"
+                                               value="<%=client.getAuthorizationCodeValidityPeriod()%>"/>
```

Comment:
Here we should get the expiry time from application. Check how the expiry for a user access token is taken?